### PR TITLE
WIP: Restore SPC f s in ipython notebook layer.

### PR DESCRIPTION
**Pull request by cpaulik**
Fix https://github.com/syl20bnr/spacemacs/issues/5671

### CODE REVIEW

File: layers/+lang/ipython-notebook/packages.el [hunk 1]
```diff
@@ -68,7 +68,9 @@
         "+" 'ein:notebook-worksheet-insert-next
         "-" 'ein:notebook-worksheet-delete
         "x" 'ein:notebook-close
-        "u" 'ein:worksheet-change-cell-type
+        "u" 'ein:worksheet-change-cell-type)
+
+      (evil-leader/set-key-for-mode 'ein:notebook-multilang-mode
```
Comment (bmag) [inline layers/+lang/ipython-notebook/packages.el]: shouldn't it be `spacemacs/set-leader-keys-for-major-mode`? (the same as the key-bindings above)

Comment (cpaulik) [inline layers/+lang/ipython-notebook/packages.el]: No. Because then the keybinding would be `SPC m f s`

Comment (bmag) [inline layers/+lang/ipython-notebook/packages.el]: Did you test it? The docstring for `evil-leader/set-key-for-mode` tells me that
> `evil-leader/set-key-for-mode` is an alias for `spacemacs/set-leader-keys-for-major-mode` in `core-keybindings.el`.

Comment (cpaulik) [inline layers/+lang/ipython-notebook/packages.el]: Ups, I did this in a hurry. It seems that there no longer is a function that can do what we need here. I don't have time right now to dive into the spacemacs keybinding logic to set this up so it works with all the editing modes.

**Comment by cpaulik** [comment]
Now I understand. The way `evil-leader/set-key-for-mode` works was changed in https://github.com/syl20bnr/spacemacs/commit/ee6a486d4cf837cfd340b8fd4715ce92cce95737 . That's why I thought we could just use it as before. Anyway I don't know enough about keymaps to change `spacemacs//init-leader-mode-map` in a way that would allow the behaviour we want here. Closing for now.

**Pull request closed without merging**